services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/ApplicationConfigurationV2PublicationConfig.java

```diff
@@ -534,6 +534,8 @@ private DGCParameters buildDgcParameters(
         .build();
     return DGCParameters.newBuilder()
         .setTestCertificateParameters(testCertificateParameters)
+        .setExpirationThresholdInDays(distributionServiceConfig.getAppConfigParameters()
+            .getDgcParameters().getExpirationThresholdInDays())
```

Comment:
Is it possible that ExpirationThresholdInDays will be null? I remember that for some other properties we have a null check ( at least for local stats)